src/api/services/slack_notify/notification.py

```
@@ -0,0 +1,44 @@
+import os
+import logging
+
+from slackclient import SlackClient
+
+
+class SlackNotification(object):
+
+    def __init__(self):
+        slack_token = os.getenv('SLACK_API_TOKEN')
+        if slack_token:
```

What happens if this doesn't exist?

---

This affirms the assignment of the `TOKEN` to `slack_token`, which has been set above. So this will always exist

---

you could add an error handler just in case it's not set up.

---

I have added a `pass` in case the `slack_token` variable is missed.

---

It's not good practice to silently ignore errors. If you're 100% sure that the token will exist, then get rid of the `if` statement.

---

It's not good practice to silently ignore errors. If you're 100% sure that `slack_token` will never be empty, then get rid of the `if` block

---

It's not good practice to silently ignore errors. If you're 100% sure that `slack_token` will never be empty, then get rid of the `if` block